### ISSUE TITLE
Suggestion: Compatibility with Overweight Farming

### ISSUE THREAD

**Issue by inkdear** [AgriCraft/AgriCraft#1466]
https://www.curseforge.com/minecraft/mc-mods/overweight-farming

This is a mod which adds a chance for some crops to spawn huge block versions instead of their normal full grown stage.

The immediate sort of compatibility that comes to mind would be the ability to configure any crop to be able to randomly grow into any arbitrary block, though perhaps only when grown off crop sticks (so the player can choose to prevent the behaviour if they're trying to gather seeds.)

**Comment by darthvader45** [comment]
Good idea.  I'm actually curious about this mod, might take it into account for a survival mod setup.
